_includes/_solo_boinc_intro.htm

```diff
@@ -5,12 +5,12 @@
         <div class="row">
             <div class="col-12 col-sm-8">
                 <h4 class="text-bold">IMPORTANT NOTICE</h4>
-                <p><b>To solo mine, you must have enough Gridcoin to <a href="/wiki/staking.html">stake</a> and earn rewards. You should have enough Gridcoin to stake about once every month, which is around 5000 GRC. Consider following the <a href="/guides/pool.htm">pool mining guide</a> if you do not have 5000 GRC</b></p>
+                <p><b>You will need approx 1-2 GRC to get started with solo crunching, which you can obtain from a faucet, a friendly user on our <a href="https://discord.gg/jf9XX4a">discord server</a>, or an exchange. To collect accrued rewards, you must <a href="/wiki/staking.html">stake</a> a block or use MRC (manual reward claim) to have somebody stake for you.  MRC does have some <a href="/wiki/faq.html">fees</a>, and the less often you make MRC requests the lower the fees will be. Once you have sufficient GRC (approx 5,000 to stake once a month on <i>average</i>), you will no longer need to make MRC claims as your wallet will be able to regularly stake blocks on its own.</p>
```

Comment:
We probably can just remove this important notice bit and move the information to other sections. Getting 1-2 GRC is not all that much so I'm not sure it needs to have a big banner about it

Reply:
The information about MRC vs regular staking should probably be included somewhere fairly early on in the process, is there somewhere else you think it would make more sense aside from the bold warning at the start?

Reply:
* and beacon cost.